specification/customer-insights/resource-manager/Microsoft.CustomerInsights/2017-01-01/customer-insights.json

```diff
@@ -457,24 +502,26 @@
         ],
         "responses": {
           "200": {
-            "description": "OK. Successfully get the profile.",
+            "description": "OK. Successfully get the segment in the hub.",
             "schema": {
-              "$ref": "#/definitions/ProfileResourceFormat"
+              "$ref": "#/definitions/SegmentResourceFormat"
             }
           }
         }
-      },
-      "delete": {
+      }
+	},
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}": {
```

Comment:
Feedback about this API -
hubBillingInfo - why are you not using the Sku property for this as defined in the ARM RPC.
Mark read only properties appropriately. Add allowed values for provisioningState property